app-office/homebank/homebank-5.6.1.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="http://homebank.free.fr/public/sources/${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 IUSE="+ofx"
-KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+KEYWORDS="amd64 ~ppc ~ppc64 ~x86"
```

Comment:
Please let the arch teams stabilize for every arch. We don't let users stabilize packages, as stabilization involves a "strict" process and we can't be sure you've followed it. 

Let me know if you need help filing a stable request bug. I'll test the version bump and commit that separately if it's fine.